.github/workflows/rollback-prod.yaml

```diff
@@ -0,0 +1,22 @@
+name: Rollback-prod
```

Comment:
We don't have hyphens in the name of our other workflows. Lets do `Rollback production` and `Rollback staging`

Reply:
`staging` or `stage`? As I observed, we are using `stage` more often

Reply:
Either

Reply:
Ok, so I choose `stage` for now, because it will fit to environment name.